### PR TITLE
INVGOV-10473 Add /blogs

### PR DESCRIPTION
**Pull request by sam-ludlow**

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -1682,6 +1682,163 @@ paths:
...
+            type: string
+            example: '2012-12-25'
+        - name: updated_date
+          description: 'Filter on date updated. Format: YYYY-MM-DD.'
+          in: query
+          schema:
+            type: string
+            example: '2012-12-25'
+        - name: status
+          description: 'Filter the list of blogs on a comma seperated list of statuses. Valid statuses are "published" and "unpublished", you may inlude both. The default when not supplied is "published" only.'
```
Comment (ParryLuke) [inline api.swagger.yaml]: the parameter is named status but the property is a boolean named 'published'

Comment (ParryLuke) [reply api.swagger.yaml]: I think we should make it consistent i.e. 'state'

File: api.swagger.yaml [hunk 2]
```diff
@@ -6560,6 +6717,165 @@ components:
...
+      type: object
+      properties:
+        title:
+          description: 'Title of the blog.'
+          type: string
+          example: 'Guidelines for creating a strong password'
+        body:
+          description: 'The body of the blog.'
+          type: string
+          example: 'You can create strong passwords using an online random password generator.'
```
Comment (jmorahan) [inline api.swagger.yaml]:
```suggestion
          example: 'You can create strong passwords using an offline random password generator.'
```
sorry, couldn't resist

**Comment by MireilleHelle** [comment]
This will be merged in https://github.com/invotra/api/pull/219/files (one overall PR for all 9 endpoints, otherwise, there will be conflicts to resolve).

**Pull request closed without merging**